tests/test_meshtastic_utils_connect_paths.py

```
@@ -39,6 +40,86 @@ def test_connect_meshtastic_returns_existing_client(reset_meshtastic_globals):
+        second.join(timeout=1.0)
+
+    assert not first.is_alive()
+    assert not second.is_alive()
+    assert worker_errors == []
+    assert worker_results[0] is created_client
+    assert worker_results[1] is created_client
+    assert mock_tcp.call_count == 1
+
+
+def test_connect_meshtastic_waiter_times_out_when_attempt_stuck(
+    reset_meshtastic_globals,
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)